src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java

```diff
@@ -355,6 +355,14 @@ public void onFailure(Exception e) {
                 listener.onFailure(new ElasticsearchSecurityException(err, RestStatus.FORBIDDEN));
                 return;
             }
+        } catch (ElasticsearchException e) {
+            if (task != null) {
```

Comment:
nit: please follow code style of the file. Even though we don't enforce code style across files in the project, it will be better to keep it the same at least within file/function. In this method `} else {` should be on the same line as `}`.

Reply:
Done.